src/components/ecosystems/Collections/CollectionsContainer.jsx

```diff
@@ -185,6 +186,31 @@ export default compose(
           })
       }
     },
+    duplicateItem: ({ collectionsApiRoute, jekyll, branch, match, activeItem, updateCachedTree, history, collectionsRoute }) => () => {
+      const { collection } = match['params']
+    
+      if (collection && branch) {
+    
+        return jekyll.collection(collection, { ref: branch })
+          .then(collection => {
+            const duplicatedItem = activeItem
+            duplicatedItem['name'] = `${duplicatedItem['name'].split('.')[0]}-copy.md`
```

Comment:
@heislercreative a period is still valid in a filename, but this logic will only pull the first part.

If filename has a date or something like `filename-1.1.2019.md`, then we'd lose everything after the first dot.

Also, how can we be sure the `-copy.md` renamed file doesn't already exist? This is the recursive check of existing filenames that we spoke about.

Reply:
@5290charlie 1) Switched to slice approach (removing last 3 characters ".md") instead of split.